astropy/wcs/src/wcslib_wrap.c

```diff
@@ -3773,38 +3893,34 @@ PyWcsprm_set_velref(
   return set_int("velref", value, &self->x.velref);
 }
 
-/* static PyObject* */
-/* PyWcsprm_get_wtb( */
-/*     PyWcsprm* self, */
-/*     /\*@unused@*\/ void* closure) { */
 
-/*   PyObject* result; */
-/*   PyObject* subresult; */
-/*   int i, nwtb; */
+static PyObject* PyWcsprm_get_wtb(PyWcsprm* self, void* closure) {
+  PyObject* list;
+  PyObject* elem;
+  int i, nwtb;
 
-/*   nwtb = self->x.nwtb; */
+  nwtb = self->x.nwtb;
+
+  list = PyList_New(nwtb);
+  if (list == NULL) {
+    Py_DECREF(list);
```

Comment:
This needs to be in the `if (elem == NULL)` branch in the loop.

Reply:
Ah, this is an unfortunate bug introduced in the last commit. However, I am not sure what does

Reply:
Inside the loop you check if `elem == NULL`, in that case you return `NULL`. Before the return there you need to use the `PyDECREF(list)`.

Reply:
oopsie... I put the decref in a wrong place. Thanks!